### DISCUSSION TITLE
change color button in AP mode to save credentials

### DISCUSSION THREAD

**Discussion by MattiaCC93** [khoih-prog/ESP_WiFiManager#92] [Q&A] [answered]
Hello,
I would like to **change the color** of the button that is generated by the ESP when it goes into AP mode and allows you to save the wifi configurations. 

Where can I find that html (or CSS) code?

Thank you

**Comment by khoih-prog** [comment, marked as answer]
Analyze and modify this global `style` line (hint: **color**)

https://github.com/khoih-prog/ESP_WiFiManager/blob/52021e9f282fb40dfbed8b23c59fd5727f7e92e8/src/ESP_WiFiManager.hpp#L199

If you're doing OK, and have time to waste, then try to modify the `html` to change the color of any individual item, if you'd like.
